Example/FramezillaExample/ViewController.swift

```diff
@@ -22,6 +22,10 @@ class ViewController: UIViewController {
     let label2 = UILabel()
     let label3 = UILabel()
 
+    let layer1 = CALayer()
+    let layer2 = CALayer()
+    let layer3 = CALayer()
```

Comment:
We can configure these layers (and all other subviews) as `lazy var`. It will allow us to reduce `viewDidLoad`